src/Commands/SetHaySqueeze.h

```diff
@@ -0,0 +1,21 @@
+#ifndef SetHaySqueeze_H
+#define SetHaySqueeze_H
+
+#include "Commands/Command.h"
+#include "../Robot.h"
+
+class SetHaySqueeze: public Command
+{
+public:
+	SetHaySqueeze(Elevator::HaySqueezeValue value);
+	void Initialize();
+	void Execute();
+	bool IsFinished();
+	void End();
+	void Interrupted();
+
+private:
+	Elevator::HaySqueezeValue mValue;
```

Comment:
By this point you should know where this is going... Consider adding Elevator.h to the includes